docs/shop_administration/product_variants_and_assets.md

```diff
@@ -0,0 +1,78 @@
+---
+description: You can create multiple variants of one product, based on attributes such as size, color, technical specifications, and others.
+---
+
+# Product variants and assets
+
+## Product variants
+
+Product variants enable you to have multiple versions of one product, differing in some characteristics.
+
+Typical examples are a t-shirt in different colors, or a laptop with different hard disk sizes.
+
+You can create variants based on product attributes.
+To use an attribute to create variants, mark the attribute as **Used for product variants** when creating a product type.
+
+![Product type definition with attributes used for variants](img/product_type_attr_used_for_variants.png)
+
+If a product has one attribute that can be used for variants, you can create variants for this attribute.
```

Comment:
```suggestion
```
already explained above, so I would skip repetition

Reply:
I wanted to contrast here: one attribute -> plain variants, multiple attributes: variants for all combinations. Maybe clarify to "only one attribute"?